proto/oracle/v1/oracle.proto

```diff
@@ -5,20 +5,27 @@ import "gogoproto/gogo.proto";
 
 option go_package = "github.com/peggyjv/sommelier/x/oracle/types";
 
-
-// UniswapData is an implementation of OracleData
+// UniswapData represents the necessary data for a given uniswap pair.
 message UniswapData {
-	repeated UniswapPair pairs = 1 [
-		(gogoproto.nullable) = false
+	string        id   = 1;
```

Comment:
So this is used to also parse the JSON from the graph. This will break the app in a number of places.

Reply:
could easily make a to parse tho...
```go
type Uniswap struct {
  Pairs []UniswapPair `json:"data"`
}
```